### ISSUE TITLE
Date Picker

### ISSUE THREAD

**Issue by emKaroly** [itchyny/calendar.vim#136]
Is there a way to copy the selected date or time and use the calendar as date picker?

**Comment by itchyny** [comment]
No. I'm afraid I don't understand the situation you want to copy the date. Also I would copy the text using the copy mode of tmux.

**Comment by emKaroly** [comment]
For example i am looking at calendar where i have some free time and navigate to that spot with cursor, than i want to copy the date an time of that spot to put that in mail or other place.

**Comment by itchyny** [comment]
Okay, I think this is worth implementation. I wonder which key sequence would be natural to trigger this feature...

**Comment by emKaroly** [comment]
Yank would be nice with some "Text object" functionality like:
"yiw" => 2018-03-22 10:00
"yw" => 2018-03-22 10:38-11:00
"yaw" => 2018-03-22 10:00-11:00
"Y" => 2018-03-22
